src/gui/updater/ocupdater.cpp

```diff
@@ -104,15 +104,23 @@ bool OCUpdater::performUpdate()
     QString updateFile = settings.value(updateAvailableC).toString();
     if (!updateFile.isEmpty() && QFile(updateFile).exists()
         && !updateSucceeded() /* Someone might have run the updater manually between restarts */) {
-        const QString name = Theme::instance()->appNameGUI();
-        if (QMessageBox::information(nullptr, tr("New %1 Update Ready").arg(name),
-                tr("A new update for %1 is about to be installed. The updater may ask\n"
-                   "for additional privileges during the process.")
-                    .arg(name),
-                QMessageBox::Ok)) {
+        const auto messageBoxStartInstaller = new QMessageBox(QMessageBox::Information,
+            tr("New %1 Update Ready").arg(Theme::instance()->appNameGUI()),
+            tr("A new update for %1 is about to be installed. The updater may ask\n"
```

Comment:
Do we really need that new line character? I think the gui should figure out line breaks on it own. If it's just because you don't want to have a long string in the source code, you can use C++ multiline strings :)

Reply:
@FlexW Not sure. I haven't checked if that newline character is not needed. Just copy-pasted that code.